packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -2142,7 +2142,7 @@ describe('shallow', () => {
       expect(wrapper.find('.bar')).to.have.lengthOf(1);
     });
 
-    it.skip('allows setState inside of componentDidMount', () => {
+    it('allows setState inside of componentDidMount', () => {
```

Comment:
should not allow setState inside of componentDidMount

Reply:
Why would it not?